src/main/java/org/example/MessageService.java

```diff
@@ -2,6 +2,7 @@
 
 public interface MessageService {
   public String getHello() ;
```

Comment:
```suggestion
  String getHello() ;
```
Interface methods are always public.

Reply:
`public String getHello() ;
 public long curiosityDateConversion(String date);
   public String getTodayDate();`
Thank you next time onword I will take care about that.